models/Document/Editable/Select.php

```diff
@@ -66,7 +66,9 @@ public function frontend()
      */
     public function setDataFromResource(mixed $data): static
     {
-        $this->text = $data;
+        if(is_string($data)) {
```

Comment:
what if data is a integer value (or float) ... should we not just cast at this place?

Reply:
Removed the check, but added the string cast